design/package-logo.svg

```diff
@@ -0,0 +1,374 @@
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     inkscape:label="Layer 1"
+     inkscape:groupmode="layer"
+     id="layer1"
+     transform="translate(0,-229.26665)">
+    <g
+       id="g4522"
+       style="stroke:#b03931;stroke-opacity:1;stroke-width:3.175;stroke-miterlimit:4;stroke-dasharray:none">
+      <g
+         id="g4536">
+        <g
+           id="g4531" />
+      </g>
+    </g>
+    <g
+       id="g4527"
+       style="stroke:#98312a;stroke-width:6.3499999;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+       transform="matrix(1.1604086,0,0,1.1604086,-5.4325038,-42.208842)">
+      <path
+         sodipodi:nodetypes="ccccccc"
+         inkscape:connector-curvature="0"
+         id="path4508"
+         d="m 8.4666666,246.19998 v 33.86667 l 25.3999994,8.46667 25.4,-8.46667 v -33.86667 l -25.4,-8.46666 z"
+         style="fill:none;fill-rule:evenodd;stroke:#98312a;stroke-width:6.3499999;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
+    </g>
+    <path
+       style="fill:#b03931;fill-opacity:1;fill-rule:evenodd;stroke:#b03931;stroke-width:8.18729782;stroke-linecap:butt;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+       d="m 33.866754,254.94601 v 32.74919 l 24.561894,-8.1873 V 246.7587 L 33.866754,238.57142 9.3048615,246.7587 Z"
+       id="path4538"
+       inkscape:connector-curvature="0"
+       sodipodi:nodetypes="ccccccc" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:40.21941757px;line-height:53.62588501px;font-family:Roboto;-inkscape-font-specification:Roboto;text-align:start;letter-spacing:0px;word-spacing:0px;text-anchor:start;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:2.6812942px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       x="36.843479"
+       y="296.21661"
+       id="text6109"
+       transform="skewY(-17.337588)"><tspan
+         sodipodi:role="line"
+         id="tspan6107"
+         x="36.843479"
+         y="296.21661"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Bold';fill:#ffffff;stroke-width:2.6812942px">D</tspan></text>
+  </g>
+  <g
+     inkscape:groupmode="layer"
+     id="layer2"
+     inkscape:label="Layer 2"
+     sodipodi:insensitive="true"
+     style="display:none">
+    <image
+       transform="translate(0,-229.26665)"
+       y="237.1985"
+       x="10.739017"
+       id="image5884"
+       xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAg0AAAIJCAIAAACp65nVAAAAA3NCSVQICAjb4U/gAAAgAElEQVR4
+nO3dyW8beX7//6rivkmkREryItNLe2u7DTc83ZPpZIJJgEGOQc7Jf5BzECT/RoCcghxyCJIAOc0E
```

Comment:
Why do you use inline png here?

Reply:
uhh, dunno what inkscape did there. I had the D logo in there once but it didn't look so good, maybe it still kept it